src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -598,6 +598,19 @@ func (c controller) sendCompleteInstallation(isSuccess bool, errorInfo string) {
 	c.log.Infof("Done complete installation step")
 }
 
+// logClusterOperatorsStatus logging cluster operators status
+func (c controller) logClusterOperatorsStatus() {
+	operators, err := c.kc.ListClusterOperators()
```

Comment:
Maybe describe as well?

Reply:
we decided to start with list, looks like we have all relevant info, if not will add more

Reply:
We get the status which is the relevant part of what you get in describe